frontend/src/scenes/trends/LineGraph.js

```diff
@@ -188,6 +188,12 @@ export function LineGraph({
                                       if (entityData.dotted && !(tooltipItem.index === entityData.data.length - 1))
                                           return null
                                       var label = entityData.chartLabel || entityData.label || ''
+                                      if (entityData.action) {
+                                          let math = entityData.action.math
```

Comment:
Thanks for this! One more thing to resolve is that we default the math to total so there are instances where `entityData.action.math` is null and not necessarily 'total'.

Reply:
Oh okay! I will update it.